src/MeshlessFV/MeshlessFVTree.cpp

```diff
@@ -578,7 +578,7 @@ void MeshlessFVTree<ndim,ParticleType>::UpdateGodunovFluxes
             neibmanager.GetParticleNeib(activepart[j],hydromask,do_pair_once);
 
 #if defined(VERIFY_ALL)
-        neibmanager.VerifyNeighbourList(i, Nhydro, mfvdata, "all");
+        neibmanager.VerifyNeighbourList(i, mfv->Nhydro, mfvdata, "all");
```

Comment:
Not sure what you mean.  This is in the active loop and i = activelist[j] already.  Or do you mean replace i with activelist[j]?